SofaKernel/modules/SofaSimpleFem/src/SofaSimpleFem/HexahedronFEMForceField.inl

```diff
@@ -833,11 +833,9 @@ void HexahedronFEMForceField<DataTypes>::computeRotationLarge( Transformation &r
     edgex.normalize();
     edgey.normalize();
 
-    Coord edgez = cross( edgex, edgey );
-    edgez.normalize();
+    const Coord edgez = cross( edgex, edgey );
```

Comment:
edgez = edgex ∧ edgey  = ‖ edgex  ‖ × ‖ edgey   ‖ × sin ⁡ ( θ ) × c   (c is unitary in edgez  direction) 
so edgez might not be always normalized if the hexahedron is deformed. However as the tests are all green this need further investigations!

Reply:
I considered your remark. Let me know what you think